autosklearn/evaluation/__init__.py

```diff
@@ -35,13 +33,28 @@ def fit_predict_try_except_decorator(ta, queue, **kwargs):
         exception_traceback = traceback.format_exc()
         error_message = repr(e)
 
-        queue.put({'loss': WORST_POSSIBLE_RESULT,
+        queue.put({'loss': cost_for_crash,
                    'additional_run_info': {'traceback': exception_traceback,
                                            'error': error_message},
                    'status': StatusType.CRASHED,
                    'final_queue_element': True})
 
 
+def get_cost_of_crash(metric):
+
+    # For metrics like accuracy that are bounded to [0,1]
+    # assume 1 as the worst cost as autosklearn works with err
+    worst_possible_result = 1.0
+
+    if metric is not None:
+        # Else, we use the make_scorer info to define the
+        # worst possible result
+        if metric._sign < 0:
+            worst_possible_result = np.iinfo(np.uint32).max
```

Comment:
As there every metric has the attribute `optimum`, I'm actually wondering whether we should add a new attribute `worst_possible_value` to each metric so that it's clearly defined (and only in one place, namely the metrics module) how they work. What do you think?